vomsimporter.py

```diff
@@ -542,6 +577,8 @@ def import_voms_user(self, voms_user):
             new_user = True
 
         self.synchronise_aup(iam_user, voms_user)
+        if self._synchronize_activation_status and iam_user['active'] == voms_user['suspended']:
```

Comment:
Nitpicking: for uniformity, can you please use the same spelling for synchroniSation as in the existing code? if you prefer, you can change the existing code, which would make it indeed more uniform since we have authZ.